bodies/views.py

```diff
@@ -88,6 +90,22 @@ def follow(self, request, pk):
 
         return Response(status=204)
 
+    def get_events(self, request, pk):
+        """Get all events from pk uuid or strid.
+        {?archived} is optional arguement to fetch old events"""
+        body = self.get_body(pk)
+        # Get query param
+        archived = request.GET.get("archived")
+        if archived is None:
+            archived = '0'
+
+        if archived not in ('0', '1'):
+            return Response({"message": "Invalid archived parameter"}, status=400)
```

Comment:
Just `archived = 'archived' in request.GET` should work I think, and you can then use that boolean in the fiter